README.md

```diff
@@ -21,7 +21,7 @@ import { View3D, Box } from '@standard/view';
 </View3D>
 ```
 
-<img src="https://github.com/standard-ai/standard-view/blob/master/example-gifs/example.png?raw=true" width="250" height="200" />
```

Comment:
I changed these to the absolute path so the npm page can actually get to the assets.
https://www.npmjs.com/package/@standard/view